stix_shifter_modules/reversinglabs/stix_translation/results_translator.py

```diff
@@ -0,0 +1,352 @@
+from stix_shifter_utils.modules.base.stix_translation.base_results_translator import BaseResultTranslator
+from . import sdo_translator
+from os import path
+import json
+from ipaddress import ip_network, IPv4Network
+from stix_shifter_utils.utils import logger
+import uuid
+
+class ResultsTranslator(BaseResultTranslator):
+
+  # Get indicator_types value
+  def get_indicator_types(self, value):
+    if value in ["KNOWN", 'known']:
+      return 'benign'
+    elif value in ['suspicious', 'SUSPICIOUS']:
+      return 'anomalous-activity'
+    elif value in ['malicious', 'MALICIOUS']:
+      return 'malicious-activity'
+    # if key/value pair is not included in the XFE report
+    elif value is None:
+      return
+    else:
+      return 'unknown'
+
+  def find_hash_type_by_length(self, value):
```

Comment:
This would be a good candidate to move to `stix_shifter_utils/utils/helpers.py`

Reply:
We can look into it on a later PR